crates/cubecl-linalg/src/matmul/tests/cmma_matmul/matmul_test_launcher.rs

```diff
@@ -104,6 +104,7 @@ pub fn test_matmul_algorithm<A, EG, ES, R>(
     if A::check_availability::<R, (EG, ES, f32)>(&client, &config).is_err() {
         // Can't execute the test.
         println!("Skipped - not supported!");
+        client.flush();
```

Comment:
shouldn't be needed anymore, but isn't a big deal.

Reply:
The PR for that isn't merged yet so I'd leave it for now so tests can actually be run.